core/src/test/java/io/questdb/griffin/O3PartitionPurgeTest.java

```diff
@@ -460,7 +497,10 @@ public void testPartitionDeletedAsyncAfterDroppedBySql() throws Exception {
                 runPartitionPurgeJobs();
 
                 path.of(engine.getConfiguration().getRoot()).concat("tbl").concat("1970-01-09").concat("x.d").$();
-                Assert.assertFalse(Chars.toString(path), Files.exists(path));
+
+                // We cannot remove the partition, it can be the version which TableWriter writes
+                // Ideally we want a way to say it's safe to delete dropped partition 1970-01-09 but there is no way atm
+                // Assert.assertFalse(Chars.toString(path), Files.exists(path));
```

Comment:
this is a serious flaw, we now leave dropped partitions behind. Lets discuss

Reply:
Ok, this is no longer the case, assert is back and dropped partition are deleted in `O3PartitionPurgeJob`